rubix/galaxy/input_handler/nihao.py

```diff
@@ -0,0 +1,155 @@
+from .base import BaseHandler
+import pynbody
```

Comment:
why do we need pynbody here? is it necessary to load the Nihao data? if not, I would rather not introduce it as a dependency